README.md

```diff
@@ -46,45 +44,48 @@ multibase comes in. It answers the question:
 The Format is:
 
 ```
-<base-encoding-character><base-encoded-data>
+<base-encoding-code-point><base-encoded-data>
 ```
 
-Where `<base-encoding-character>` is used according to the multibase table.
+Where `<base-encoding-code-point>` is a code representing an entry in the multibase table.
 
 ### Multibase Table
 
 The current multibase table is [here](multibase.csv):
 
 ```
-encoding,          code, description,                                                  status
-identity,          0x00, 8-bit binary (encoder and decoder keeps data unmodified),     default
-base2,             0,    Binary (01010101),                                            candidate
-base8,             7,    Octal,                                                        draft
-base10,            9,    Decimal,                                                      draft
-base16,            f,    Hexadecimal (lowercase),                                      default
-base16upper,       F,    Hexadecimal (uppercase),                                      default
-base32hex,         v,    RFC4648 case-insensitive - no padding - highest char,         candidate
-base32hexupper,    V,    RFC4648 case-insensitive - no padding - highest char,         candidate
-base32hexpad,      t,    RFC4648 case-insensitive - with padding,                      candidate
-base32hexpadupper, T,    RFC4648 case-insensitive - with padding,                      candidate
-base32,            b,    RFC4648 case-insensitive - no padding,                        default
-base32upper,       B,    RFC4648 case-insensitive - no padding,                        default
-base32pad,         c,    RFC4648 case-insensitive - with padding,                      candidate
-base32padupper,    C,    RFC4648 case-insensitive - with padding,                      candidate
-base32z,           h,    z-base-32 (used by Tahoe-LAFS),                               draft
-base36,            k,    Base36 [0-9a-z] case-insensitive - no padding,                draft
-base36upper,       K,    Base36 [0-9a-z] case-insensitive - no padding,                draft
-base58btc,         z,    Base58 bitcoin,                                               default
-base58flickr,      Z,    Base58 flicker,                                               candidate
-base64,            m,    RFC4648 no padding,                                           default
-base64pad,         M,    RFC4648 with padding - MIME encoding,                         candidate
-base64url,         u,    RFC4648 no padding,                                           default
-base64urlpad,      U,    RFC4648 with padding,                                         default
-proquint,          p,    Proquint (https://arxiv.org/html/0901.4016),                  draft
-base256emoji,      🚀,    Base256 with custom alphabet using variable-sized-codepoints, draft
+Unicode,    character,  encoding,           description,                                                    status
```

Comment:
The previous commit changed the description column a bit (also the casing), I think it would make sense to use those change and not switching back to the old one.

Reply:
Oh man, missed that in the merge conflict 🤦‍♀️